benchexec/tools/libkluzzer.py

```diff
@@ -21,11 +21,20 @@ def program_files(self, executable):
             executable, self.REQUIRED_PATHS, parent_dir=True
         )
 
-    def executable(self):
-        return util.find_executable("LibKluzzer", "bin/LibKluzzer")
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("LibKluzzer", subdir="bin")
 
     def version(self, executable):
         return self._version_from_tool(executable)
 
     def name(self):
         return "LibKluzzer"
+
+    def cmdline(self, executable, options, task, rlimits):
+        data_model_param = get_data_model_from_task(
+            task, {"ILP32": "--32", "LP64": "--64"}
+        )
+        if data_model_param and data_model_param not in options:
+            options += [data_model_param]
+
+        return super().cmdline(executable, options, task, rlimits)
```

Comment:
Same as above.

Reply:
fixed in 4b7464cc40a620880e02a86218a0eb1d0b26ac8c